From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Fri, 18 Dec 2020 23:26:16 +0100
Subject: [PATCH] github workflow: run apt-get update

E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/g/glibc/libc6-dbg_2.27-3ubuntu1.3_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 .github/workflows/build-and-unittest.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.github/workflows/build-and-unittest.yaml b/.github/workflows/build-and-unittest.yaml
index 929f63a6..389578be 100644
--- a/.github/workflows/build-and-unittest.yaml
+++ b/.github/workflows/build-and-unittest.yaml
@@ -13,6 +13,8 @@ jobs:
         run: echo ZRAM=$(sudo cat /sys/class/zram-control/hot_add) >> $GITHUB_ENV
       - name: set-zram-size
         run: echo 1G | sudo tee /sys/block/zram$ZRAM/disksize
+      - name: update
+        run: sudo apt-get update
       - name: dependencies
         run: >
           sudo apt-get install --yes gcc
-- 
2.17.2

